app/controllers/providers/allocations_controller.rb

```diff
@@ -45,11 +45,9 @@ def show
     end
 
     def initial_request
-      get_training_providers_without_previous_allocations
+      flow = InitialRequestFlow.new(params: params)
```

Comment:
`get_training_providers_without_previous_allocations` can be removed?

Reply:
yep, since this has been extracted out